src/CSS/Fonts.css

```diff
@@ -50,47 +50,76 @@
 }
 
 @font-face {
-  font-family: 'Lexend';
+  font-family: 'Lexend-Light';
   src: url('../Fonts/Lexend-Light.ttf') format('truetype');
   font-weight: 300;
   font-style: normal;
 }
 
 @font-face {
-  font-family: 'Lexend';
+  font-family: 'Lexend-ExtraThin';
```

Comment:
Technically seems to be `ExtraLight`, not `ExtraThin` - maybe worth changing, to avoid any confusion?

Btw: Cool to know how this is supposed to be done properly ^^ Thanks @emgerold